wp/Makefile

```diff
@@ -1,25 +1,63 @@
-BAP_WP = lib/bap_wp
-WP = plugin
-SAMPLE_BINS = resources/sample_binaries
+SHELL=/bin/bash
+LIB := lib/bap_wp
+PLUGIN := plugin
+SAMPLE_BINS := resources/sample_binaries
+FMT_STR := "\u001b[31m%s\u001b[0m\n"
 
-.PHONY: install doc test clean reinstall
+.PHONY: test.lib test.plugin test all clean
```

Comment:
I think basically all the targets in this file should be marked phony?

Reply:
Agreed and changed!